treadmill-aws/sbin/start_zookeeper.sh

```diff
@@ -98,6 +98,8 @@ ${DISTRO}/bin/treadmill \
         --master-id ${TREADMILL_ZOOKEEPER_MYID} \
         --no-run
 
+ln -s ${INSTALL_DIR}/treadmill /treadmill
```

Comment:
this should not be needed. rather, for zookeeper, there is a config option that will specify where to "root" when looking for acl files.